app/src/main/res/values/constants.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="min_name_length">2</integer>
+    <integer name="max_name_length">30</integer>
+</resources>
```

Comment:
a blank line at the end of this file, please :)

Reply:
Aha! Missing blank lines at the end lint check finds another victim! 😂 
I'll add this and update the PR soon.